<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT_classifier_ClassificationClass</class>
 <widget class="QMainWindow" name="QT_classifier_ClassificationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Identify Era</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;blackboard.jpg&quot;)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="file_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>840</y>
      <width>350</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;
background-image: url(&quot;&quot;)</string>
    </property>
    <property name="text">
     <string>選擇影像檔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_editline">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>870</y>
      <width>700</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 3px solid #FFFFFF;
color: #FFFFFF</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>139</y>
      <width>350</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 3px;
border: 3px solid #FFFFFF;
color: #FFFFFF;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_image_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>75</y>
      <width>1100</width>
      <height>450</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>25535</width>
      <height>25535</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 3px;
border: 3px solid #FFFFFF;
color: #FFFFFF;
background-image: url(&quot;&quot;)</string>
    </property>
    <property name="text">
     <string>影像檔</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbfile_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>600</y>
      <width>350</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;
background-image: url(&quot;&quot;)</string>
    </property>
    <property name="text">
     <string>選擇模型檔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_pb">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>630</y>
      <width>700</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 3px solid #FFFFFF;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>730</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;
background-image: url(&quot;&quot;)</string>
    </property>
    <property name="text">
     <string>開始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>70</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFFFFF;
background-image: url(&quot;&quot;)</string>
    </property>
    <property name="text">
     <string>影像內含物名稱</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_button">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>860</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;
background-image: url(&quot;&quot;)</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QT_yolov2_Console.qrc"/>
 </resources>
 <connections/>
</ui>
